README.md

```diff
@@ -35,7 +35,7 @@ python cs_upload.py --trace-file foo.cst.gz ...
 
 - `--trace-file` (required): trace file to upload
 - `--trace-name` (optional): name of the trace created. Defaults to the file name.
-- `--project-number` (required): the number of the project to upload to. You can find this
+- `--project-id` (required): the number of the project to upload to. You can find this
   in the web UI if you navigate to that project and then look in the URL, for example in
   `https://[]/project/1022/dashboard`, the trace number is 1022.
 - `--profile-number` (optional): the number of the profile you want to use for generating a
```

Comment:
**Typo:** s/profile-number/profile-id/

You fixed `project-number`, but the same should be applied to `profile-`

Reply:
fixed